src/go/rdctl/cmd/listSettings.go

```diff
@@ -40,10 +63,63 @@ var listSettingsCmd = &cobra.Command{
 	},
 }
 
+// the reg file format is directly usable only on Windows,
+// but it can be created on any platform for purposes of testing or development
+
 func init() {
 	rootCmd.AddCommand(listSettingsCmd)
+	listSettingsCmd.Flags().StringVarP(&outputSettingsFlags.Format, "output", "", jsonFormat, fmt.Sprintf("output format: %s|%s, default %s", jsonFormat, regFormat, jsonFormat))
+	listSettingsCmd.Flags().StringVarP(&outputSettingsFlags.RegistryHive, "reg-hive", "", "", fmt.Sprintf("registry hive: %s|%s, default %s", reg.HklmRegistryHive, reg.HkcuRegistryHive, reg.HklmRegistryHive))
```

Comment:
I'd rather treat `rdctl list-settings --output=json --reg-hive=X` as an error because the output won't meet the user's expectations (whatever they might be).